frontend/src/metabase/redux/app.ts

```diff
@@ -1,15 +1,12 @@
+import { createAction } from "@reduxjs/toolkit";
```

Comment:
I would like to in a separate PR remove the `react-actions` package as RTK has everything we need. Using this method results in a return type RTK is happy with.